src/com/walmartlabs/lacinia/pedestal/subscriptions.clj

```diff
@@ -114,21 +115,24 @@
              (case type
                "connection_init"
                (when (>! response-data-ch {:type :connection_ack})
-                 (recur subs))
+                 (recur subs (assoc connection-state :initial-payload payload)))
```

Comment:
I think this might be better named `:connection-payload`.

Reply:
Currently, this code uses an entire map with a single key and merges it into the Pedestal context used to invoke the streamer.  However, there is only a single value, so perhaps the `go-loop` variable should be `connection-payload` with a default of nil, and that can be `assoc`'ed in below.